waspc/.gitignore

```diff
@@ -2,6 +2,7 @@
 stack*.yaml.lock
 dist-newstyle
 waspc.cabal
+.vscode
```

Comment:
This was triggered because I wanted to exclude yaml files from autoformatting in this project (as I noticed it created whitespace), while not wanting to stop reformatting yaml everywhere else. So I needed a local .vscode config.

Reply:
Aha well makes sense! However while that is the reason for you adding .vscode, my question from above remains, right? Should we add rules to project-level .gitignore in order to ignore the local settings of our editors, or should we instead put those rules into our machine-local gitignores? And if we do put them in project-level .gitignore, I suggested we at least group them and put a comment as a header.
What are your thoughts on that?